.travis.yml

```diff
@@ -1,25 +1,20 @@
 ---
+dist: xenial
 language: python
-python: "2.7.12"
-sudo: required
-
-env:
-  - ansible_version: 2.4.1.0
-  - ansible_version: 2.4.2.0
-  - ansible_version: 2.4.3.0
-  - ansible_version: 2.4.4.0
-  - ansible_version: 2.5.0.0
-
+python: "3.8"
+os: linux
 services:
   - docker
-
 install:
-  - pip install ansible==${ansible_version}
-  - pip install molecule
-  - pip install docker
+  - pip install pipenv
+  - pipenv sync
+env:
+  jobs:
+    - MOLECULE_DISTRO=debian:stretch-slim
 
 script:
-  - molecule test
+  - export MOLECULE_DISTRO=${MOLECULE_DISTRO}
```

Comment:
I migth be wrong, but I believe this is unnecesary since the variable is already set in the jobs block.

Reply:
Done